### PR TITLE
Add ranking suggestion to "Preserving sort"

### PR DESCRIPTION
**Pull request by ehrenMSFT**

### CODE REVIEW

File: powerquery-docs/CommonIssues.md [hunk 1]
```diff
@@ -19,10 +19,11 @@ For example, if you sort a sales table so that each store's largest sale is show
 
 Because of the way Power Query optimizes certain operations, including skipping them or offloading them to data sources (which can have their own unique ordering behavior), sort order isn't guaranteed to be preserved through aggregations (such as `Table.Group`), merges (such as `Table.NestedJoin`), or duplicate removal (such as `Table.Distinct`).
 
-There are a number of ways to work around this. Here are two suggestions:
+There are a number of ways to work around this. Here are few suggestions:
```
Comment (jwelchMS) [inline powerquery-docs/CommonIssues.md:22]: Here are **a** few suggestions.

File: powerquery-docs/CommonIssues.md [hunk 2]
```diff
@@ -19,10 +19,11 @@ For example, if you sort a sales table so that each store's largest sale is show
...
 
 * Perform a sort *after* applying the downstream operation. For example, when grouping rows, sort the nested table in each group before applying further steps. Here's some sample M code that demonstrates this approach: `Table.Group(Sales_SalesPerson, {"TerritoryID"}, {{"SortedRows", each Table.Sort(_, {"SalesYTD", Order.Descending})}})`
 * Buffer the data (using `Table.Buffer`) before applying the downstream operation. In some cases, this operation will cause the downstream operation to preserve the buffered sort order.
+* Use [ranking](/rank-column). For example, instead of using `Table.Distinct`, you could order by the column containing the duplicate values, rank based on a tie-breaker column (such as `modified_date`), and then filter to keep just the rank 1 rows.
```
Comment (jwelchMS) [inline powerquery-docs/CommonIssues.md:26]: "order by the column**(s)**" to catch the case where there are multiple columns to use for the distinct values.

**Review by jwelchMS — APPROVED** [review]

File: powerquery-docs/CommonIssues.md [hunk 3]
```diff
@@ -19,10 +19,11 @@ For example, if you sort a sales table so that each store's largest sale is show
...
 
-There are a number of ways to work around this. Here are two suggestions:
+There are a number of ways to work around this. Here are a few suggestions:
 
 * Perform a sort *after* applying the downstream operation. For example, when grouping rows, sort the nested table in each group before applying further steps. Here's some sample M code that demonstrates this approach: `Table.Group(Sales_SalesPerson, {"TerritoryID"}, {{"SortedRows", each Table.Sort(_, {"SalesYTD", Order.Descending})}})`
 * Buffer the data (using `Table.Buffer`) before applying the downstream operation. In some cases, this operation will cause the downstream operation to preserve the buffered sort order.
+* Use [ranking](/rank-column). For example, instead of using `Table.Distinct`, you could order by the column(s) containing the duplicate values, rank based on a tie-breaker column (such as `modified_date`), and then filter to keep just the rank 1 rows.
```
Comment (DougKlopfenstein) [inline powerquery-docs/CommonIssues.md:26]: The link isn't going to work - use "[ranking](rank-column.md)" instead.

**Comment by JasonWHowell** [comment]
@ehrenMSFT

Thanks for the docs contributions. For the future docs contributions - use the private repo (powerquery-docs-pr) to get a richer experience (Acrolinx and preview builds) internally. 

Thanks~ Jason

**Pull request merged**